doc/themes.rst

```diff
@@ -0,0 +1,35 @@
+How to set the themes on production instances:
+==============================================
+
+First time:
+~~~~~~~~~~~
+
+You must add this line to your local RC file:
+
+.. code:: sh
+
+    theme2_repositories__git-urls=git@github.com:bluenove/assembl2-client-themes.git
+
+
+Then you must add your public ssh key to the github repo
+
+Then run app_compile locally:
+
+.. code:: sh
+
+    fab -c local.rc app_compile
+
+By now you should have access to ``assembl/static2/css/themes/vendor/assembl2_client_themes``
+
+Create theme for a debate:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each debate has his themes documented in a separate folder. Be careful, the folder name should be exactly the name of the debate (slug).
+
+Add your themes, commit them and push them
```

Comment:
You should specify that develop must be identical to master. When you add your themes, commit and push on develop, you need to merge develop in master right after the push